omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,60 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+
+       /**
+         * Method to load Tags
+         */
+       var thisPersonUuid='';
```

Comment:
I have 2 major concerns that are have been trying to point out,
- The init function is some sort of initializer that is supposed to be called once and not several times like you are doing.
- If you have a 'global' or scoped variable, then you should never be passing it around to every call to loadTags because then why did you declare it in the first place?

Did you see my other comment about how you declare and call loadTags()?